cardigan/.storybook/main.js

```diff
@@ -49,6 +49,12 @@ module.exports = {
       },
     });
 
+    config.resolve.fallback = {
```

Comment:
> BREAKING CHANGE: webpack < 5 used to include polyfills for node.js core modules by default.
This is no longer the case. Verify if you need this module and configure a polyfill for it.